scripts/deploy.sh

```diff
@@ -197,7 +197,7 @@ else
     error $? "Could not pushd /data/socorro"
     PYTHONPATH=. ../socorro-virtualenv/bin/python \
         ../socorro-virtualenv/bin/alembic \
-        -c config/alembic.ini upgrade head &> /var/log/socorro/alembic.log
+        -c config/alembic.ini upgrade heads &> /var/log/socorro/alembic.log
```

Comment:
Yes, it's STDOUT only, since `&` assumes `&1` in Bash (hence `2>` to capture STDERR).

As for failures being captured in the log already, well, I guess that command doesn't output to STDERR at all, then. :wink:

It's.. not critical.  Just my grizzled old sysadmin brain clicking in. :grin: